server/core/configmanager.cc

```diff
@@ -157,7 +157,7 @@ std::string sql_update_status(const std::string& cluster, int64_t version, const
     return ss.str();
 }
 
-bool is_noop_chage(const mxb::Json& lhs, const mxb::Json& rhs)
+bool is_noop_change(const mxb::Json& lhs, const mxb::Json& rhs)
```

Comment:
It's possible that `chage` refers to https://linux.die.net/man/1/chage, but I don't think so...
(I'm not particularly confident)